README.md

```diff
@@ -2,7 +2,35 @@
 
 # DAZN Lambda Powertools
 
-An integrated suite of powertools for Lambda functions to make it effortless for you to comply with our guidelines around logging and monitoring:
+`dazn-lambda-powertools` is a collection of middlewares that make working with lambda easier.
+
+## Motivation
+
+Writing Lambda's often involves the bootstrapping of specific tooling, like reading and forwarding on correlation-id's, emitting logs on a lambda timeout, and more.
```

Comment:
```suggestion
Writing Lambda functions often involves the bootstrapping of specific tooling, like reading and forwarding on correlation-id's, emitting logs on a Lambda timeout, and more.
```
nitpick: technically i don't think you can pluralise Lambda

Reply:
Ah damnit, greengrocers apostrophe got me again 😆